Detectors/TPC/base/include/TPCBase/Mapper.h

```diff
@@ -53,47 +114,114 @@ class Mapper {
 //   Mapper(const Mapper&) = delete;
 //   void operator=(const Mapper&) = delete;
 
-static GlobalPosition3D LocalToGlobal(const LocalPosition3D pos, const double alpha)
-{
-  const double cs=cos(alpha), sn=sin(alpha);
-  return GlobalPosition3D(float(double(pos.getX())*cs-double(pos.getY())*sn),
+  static GlobalPosition3D LocalToGlobal(const LocalPosition3D pos, const double alpha)
+  {
+    const double cs=std::cos(alpha), sn=std::sin(alpha);
+    return GlobalPosition3D(float(double(pos.getX())*cs-double(pos.getY())*sn),
+                            float(double(pos.getX())*sn+double(pos.getY()*cs)),
+                            pos.getZ());
```

Comment:
Didn't we already have a version that uses tabulated values for cs, sn based on the sector?